bot_management/my_keys.py

```diff
@@ -0,0 +1,19 @@
+from binance.client import Client
+
```

Comment:
As mentioned earlier, we should not store any apikey in our codebase.

Reply:
it isn't used bu users. only used in tests ant to se if code works before i can connect to rest api